composed_api.rst

```diff
@@ -386,14 +485,31 @@ Example
 
 ::
 
-    curl -u <username> http://gridte.rsgis.erdc.dren.mil/te_ba/api/v1/export/1335?source=grid
+    curl -u <username> http://gridte.rsgis.erdc.dren.mil/te_ba/api/v2/export/1335/?source=grid
 
 ::
 
     {
-      "GRiD API": {
-        "API Version": "v1"
-        }
+      "export": 
+      {
+        "status": "SUCCESS",
```

Comment:
Another comment on consistency. Elsewhere, we use `"success": true` to communicate success. Here, we use `"status": "SUCCESS"`, which makes sense as it could be in one of several possible states.

I'm starting to wonder if it makes sense to even return `"success": true` in the other instances (which makes me regret inserting the comment here). Let's consider what may cause `"success": false`. Can't we bin those reasons into client-side and server-side issues, return status code 4xx/5xx and populate the already defined Error object as needed to describe the issue. That seems more useful than just saying `"success": false`.